docs/feature-flags/sdks/client-sdks/ios.md

```diff
@@ -6,13 +6,10 @@ iOS implementation of the Eppo Randomization and Feature Flagging SDK.
 
 While in XCode:
 
-> 1. choose `File > Add Packages`
-> 2. Click `Add Local`. Which will add two targets to your project:
->
->    * eppo-flagging
->    * eppo-flagging-tests
->
->     Then select the eppo-flagging-tests target in Xcode and run Product > Test.
+> 1. Choose `Package Dependencies`
+> 2. Click `+` and enter package URL: `git@github.com:Eppo-exp/ios-sdk.git`
+> 3. Set dependency rule to `Up to Next Minor Version` and select `Add Package`
```

Comment:
`Up to Next Minor Version`

Is that reasonable?

![Screenshot 2023-01-05 at 8 31 14 PM](https://user-images.githubusercontent.com/57361/210930431-1ccaca13-6910-4bbb-9e04-d385c2e4248e.png)

Reply:
Looks great to me.